File: dart/condicional/3.dart
import 'dart:io';

void main() {
  print("Número negativo, positivo o cero");
  
  print("Digite un número positivo o negativo");
  
  int x = int.parse(stdin.readLineSync()!);

  if(x > 0){
      
      print("Su número es positivo.");
      
  }else if(x < 0){
      
      print("Su número es negativo.");
      
  }else{
      
      print("El número digitado es 0.");
      
  }
}

File: CondicionalesCsharp/3.cs
using System;


namespace Condicionales
{
    internal class Tres
    {
        public Tres() {
            Console.WriteLine("Digite un número: ");
            int num = int.Parse(Console.ReadLine());

            if (num > 0) {
                Console.WriteLine("El número digitado es positivo");
            }else if (num < 0) {
                Console.WriteLine("El número digitado es negativo");
            }
            else
            {
                Console.WriteLine("El número es 0");
            }
        }
    }
}
